client/scss/components/forms/_field-comment-control.scss

```diff
@@ -49,6 +49,14 @@
   }
 }
 
+.w-field--date_field,
+.w-field--date_time_field,
+.w-field--time_field {
+  .w-field__comment-button {
+    position: relative;
```

Comment:
Looking great @Theresa-o, this is just missing the positioning tweaks a few lines above so the button sits where intended vertically.